src/shim/Makefile

```diff
@@ -0,0 +1,71 @@
+# Root directory of the project (absolute path).
+ROOTDIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+# Base path used to install.
+DESTDIR ?= /usr/local
+
+ifneq ($(GO111MODULE),off)
+	MOD_VENDOR := "-mod=vendor"
```

Comment:
do we want/need to support vendored builds and keep `vendor/`?

Reply:
Hi, Mikko. The vendor mechanism solves the problem of reproducible construction of Go programs to a certain extent, however, its experience is not so good. The huge vendor directory needs to be submitted to the repo. Do you have any good suggestions, please? Thank you very much.

Reply:
My preference would be to not keep `vendor/` at least in the initial stage but I'm not aware of CoCo project needs to have support for offline builds etc. which might require it.

Reply:
@mythi  still have some comments to this PR?

Reply:
@YangLiang3 sorry I have not seen any updates being pushed. I will do the review shortly

Reply:
appreciate Mikko：）